driver/src/test/java/org/neo4j/driver/stress/SingleInstanceStressIT.java

```diff
@@ -83,23 +83,6 @@ List<BlockingCommand<Context>> createTestSpecificBlockingCommands() {
                 new BlockingFailingQueryInTx<>(driver));
     }
 
-    @Override
-    List<AsyncCommand<Context>> createTestSpecificAsyncCommands() {
-        return Arrays.asList(
-                new AsyncReadQuery<>(driver, false),
-                new AsyncReadQuery<>(driver, true),
-                new AsyncReadQueryInTx<>(driver, false),
-                new AsyncReadQueryInTx<>(driver, true),
-                new AsyncWriteQuery<>(this, driver, false),
-                new AsyncWriteQuery<>(this, driver, true),
-                new AsyncWriteQueryInTx<>(this, driver, false),
-                new AsyncWriteQueryInTx<>(this, driver, true),
-                new AsyncWrongQuery<>(driver),
-                new AsyncWrongQueryInTx<>(driver),
-                new AsyncFailingQuery<>(driver),
-                new AsyncFailingQueryInTx<>(driver));
-    }
```

Comment:
Is this tests not needed?

Reply:
For some reason this was not actually used. But thinking about it, we can probably give it a go. 🤔  I have actually enabled this now.